app/src/renderer/App.vue

```diff
@@ -1,12 +1,13 @@
 <template lang="pug">
 #app
-  app-header
-  #app-content
-    router-view
-    app-footer
-  notifications(:notifications='notifications' theme='cosmos')
-  modal-help
-  session
+  template(v-if="!config.modals.session.active")
```

Comment:
Cool! I like the switching of components in the container more then in the component itself. This makes it better composable.